addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/OrderReceipt.xml

```diff
@@ -82,7 +82,7 @@
                         </t>
                         <div class="pos-receipt-left-padding">
                             <t t-esc="Math.round(line.quantity * Math.pow(10, env.pos.dp['Product Unit of Measure'])) / Math.pow(10, env.pos.dp['Product Unit of Measure'])"/>
-                            <t t-if="line.unit_name !== 'Units'" t-esc="line.unit_name" />
+                            <t t-if="line.is_in_unit" t-esc="line.unit_name" />
```

Comment:
I think it should be '! line.is_in_unit', because it is when we are not in unit that we want the t-esc